index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=index, initial-scale=1.0" />
+    <title>index</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto"
+    />
+  </head>
+  <body>
+    <article class="navigation-bar">
+      <a href="https://youtube.com">
+        <img
+          style="float: left; margin-left: 23px"
+          src="Capture.JPG"
+          alt="youtube logo"
+        />
+      </a>
+      <input class="search-settings" type="text" placeholder=" Search" />
+      <img
+        style="margin-left: -4px; margin-top: 2px"
+        src="Capture1.JPG"
+        alt="Search button and other navigations"
+      />
+    </article>
```

Comment:
please use HTML5 semantic tags like nav for the navigation bar

Reply:
Good job @amexabee, but please make sure to explain why you requested the change so that the author can understand the reason behind your suggestion.